modules/src/events.rs

```diff
@@ -8,7 +8,7 @@ use serde_derive::{Deserialize, Serialize};
 use std::collections::HashMap;
 use std::convert::{TryFrom, TryInto};
 use tendermint::block;
-use tendermint::rpc::event_listener::{ResultEvent, TMEventData};
+use tendermint_rpc::client::event_listener::{ResultEvent, TMEventData};
```

Comment:
I think you don't need access to the `client` module directly but to the `event_listener` instead (as previously).

I'll try to hack together a PR to confirm this.

Reply:
I don't seem to have perms to push a branch anymore. I'll use a fork instead. Not sure CI will trigger though.